src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -10,6 +10,7 @@
 from azure.cli.core.commands.parameters import (resource_group_name_type, get_location_type,
                                                 get_resource_name_completion_list, file_type,
                                                 get_three_state_flag, get_enum_type, tags_type)
+from azure.cli.core.local_context import LocalContextAttribute, SET, GET
```

Comment:
what's SET/GET? could you pls put them into "LocalContextActions"

Reply:
I defined it as CONSTANT currently, I will define a LocalContextAction enum later. As the change is in `core`, I want to leave this work to next sprint.